packages/expo-updates/android/src/main/java/expo/modules/updates/db/entity/UpdateEntity.java

```diff
@@ -0,0 +1,56 @@
+package expo.modules.updates.db.entity;
+
+import expo.modules.updates.db.enums.UpdateStatus;
+
+import org.json.JSONObject;
+
+import java.util.Date;
+import java.util.UUID;
+
+import androidx.annotation.NonNull;
+import androidx.room.ColumnInfo;
+import androidx.room.Entity;
+import androidx.room.ForeignKey;
+import androidx.room.Index;
+import androidx.room.PrimaryKey;
+
+import static androidx.room.ForeignKey.CASCADE;
+
+@Entity(tableName = "updates",
+        foreignKeys = @ForeignKey(entity = AssetEntity.class,
+                                  parentColumns = "id",
+                                  childColumns = "launch_asset_id",
+                                  onDelete = CASCADE),
+        indices = {@Index(value = "launch_asset_id"),
+                   @Index(value = "commit_time", unique = true)})
+public class UpdateEntity {
+  @PrimaryKey
+  @ColumnInfo(typeAffinity = ColumnInfo.BLOB)
+  @NonNull
+  public UUID id;
+
+  @ColumnInfo(name = "commit_time")
+  @NonNull
+  public Date commitTime;
+
+  @ColumnInfo(name = "runtime_version")
```

Comment:
Might want to think over the name for this field so it's consistent with the verbiage glossary (but we can also change it after this commit as long as it's before publishing the package)